pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -332,11 +332,6 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 				// ReplicationManager.syncReplicationController() -> updateReplicaCount()
 				{
```

Comment:
That's true, because @liggitt complained only about get, he didn't mention update, does it apply there as well?